CHANGES/881.feature.rst

```diff
@@ -0,0 +1 @@
+Enable publishing a universal (pure Python) wheel on PyPI
```

Comment:
The change notes should use the past tense with the end-users as the target audience in mind. They should include full sentences with periods et al.
Also, "universal" is an incorrect term — conventionally, wheels tagged as `py2.py3` used to be called universal. But we should only have `py3` in the tag.